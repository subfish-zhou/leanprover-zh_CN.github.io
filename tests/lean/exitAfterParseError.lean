

def foo 

#exit

def bla
