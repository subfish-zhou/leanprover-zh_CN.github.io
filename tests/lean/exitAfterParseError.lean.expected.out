exitAfterParseError.lean:5:0: error: expected ':=', 'where' or '|'
